doc/api/next_api_changes/behavior/22135-JSS3.rst

```diff
@@ -0,0 +1,8 @@
+Figure & subfigure clearing
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Fixed a bug in which ``Figure.clf()`` and ``Figure.clear()`` were broken 
+when ``Figure`` contained axes not stored in `Figure._localaxes`.  Moved the
```

Comment:
```suggestion
when ``Figure`` contained axes not stored in ``Figure._localaxes``.  Moved the
```

This should fix the doc build. The private attribute `_localaxes` does not have a rendered documentation and thus cannot be linked to, resulting in unhappy sphinx:

> /home/circleci/project/doc/api/next_api_changes/behavior/22135-JSS3.rst:3: WARNING: py:obj reference target not found: Figure._localaxes